src/status_im/contexts/wallet/events.cljs

```diff
@@ -437,6 +437,10 @@
      (background-timer/clear-timeout current-timeout)
      {:db (assoc db :wallet/local-suggestions [] :wallet/valid-ens-or-address? false)})))
 
+(rf/reg-event-fx :wallet/clean-account-selection
+ (fn [{:keys [db]}]
+   {:db (update-in db [:wallet :ui :send] dissoc :send-account-address)}))
```

Comment:
Important step!

Recently I noticed when we navigate to a collectible details, for a moment, we show the previous displayed collectible details. that's because we are not cleaning the collectibles db when navigating back, so it's good we are cleaning these keys :+1: